ngmt/modules/ssd/_pham.py

```diff
@@ -94,28 +97,29 @@ def __init__(
         self.postural_transitions_ = None
 
     def detect(
-        self, data: pd.DataFrame, sampling_freq_Hz: float, plot_results: bool = False
+        self, data: pd.DataFrame, acceleration_data_unit: str, gyro_data_unit: str, sampling_freq_Hz: float, plot_results: bool = False
```

Comment:
Hmm, we have to really come up with a Standard Operating Procedure (SOP). For example, why do we use `acceleration_data_unit` and `gyro_data_unit`. I would say either use:

- `accel` for accelerations or accelerometer data, and 
- `gyro` for angular velocity or gyroscope data

to be in line with [Restricted keyword list for channel type](https://bids-specification.readthedocs.io/en/stable/modality-specific-files/motion.html#restricted-keyword-list-for-channel-type)

Reply:
I handled this issue as well. Now, in all algorithms, all variables are consistent with SOP.

Reply:
Do we have a SOP already??

Reply:
At the moment, we don't have any written SOP, we are using standard definitions to align with BIDS format.